utils/ci/corefile_in_gha.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+cd daos || exit 1
+
+echo ::group::Check for corefiles
+
+echo BASE_DISTRO="${BASE_DISTRO}"
+echo -n core_pattern= ; cat /proc/sys/kernel/core_pattern
+
+if [ $(echo "$BASE_DISTRO" | grep -E -q -i 'ubuntu|alma|rocky') ]
```

Comment:
(lint) Quote this to prevent word splitting. [SC2046]
(lint) Use grep -q instead of comparing output with [ -n .. ]. [SC2143]